state_machine/applications/system-check/tests/imu_test.py

```diff
@@ -0,0 +1,329 @@
+    acc, gyro, mag = request_imu_data(prompt)
+    # if x and y, test z
+    if grav_xdir and grav_ydir:
+        grav_zdir = check_gravity_acc(acc, "z")
+        if not grav_zdir:
+            # fail test
+            result_string = """X and Y direction successful in reading g m/s^2,
+failed Z."""
+            print(result_string)
+            result_dict["IMU_AccGravity"] = (result_string, False)
+            return result_dict
+    # if y and z, test x
+    elif grav_ydir and grav_zdir:
+        grav_xdir = check_gravity_acc(acc, "x")
+        if not grav_xdir:
+            # fail test
+            result_string = """Y and Z direction successful in reading g m/s^2,
+failed X."""
+            print(result_string)
+            result_dict["IMU_AccGravity"] = (result_string, False)
+            return result_dict
+    # if x and z, test y
+    elif grav_xdir and grav_zdir:
+        grav_ydir = check_gravity_acc(acc, "y")
+        if not grav_ydir:
+            # fail test
+            result_string = """X and Z direction successful in reading g m/s^2,
+failed Y."""
+            print(result_string)
+            result_dict["IMU_AccGravity"] = (result_string, False)
+            return result_dict
+
+    # if everything has passed and we haven't exited yet, update
+    result_string = """X, Y, and Z directions successful in reading g m/s^2."""
+    print(result_string)
+    result_dict["IMU_AccGravity"] = (result_string, True)
+    return result_dict
+
+
+def stationary_imu_test(result_dict):
+    """
+    Return acceleration, gyro, and magnetic readings from request_imu_data
+    If None, update result dictionary that test was not run
+    Else, check that the norms of the average accelerometer and gyroscope
+    readings are near 0 and update result dictionary
+    """
+    # prompt user and get imu data
+    prompt = "Please leave the cubesat stationary on a table.\n"
+    acc, gyro, mag = request_imu_data(prompt)
+
+    if acc is None and gyro is None:
+        result_dict["IMU_GyroStationary"] = (
+            "Stationary test not completed.", None)
+        return result_dict
+
+    # else continue running the test
+    gyro_string = f"Gyro: {tuple(gyro)} (deg/s)"
+
+    # if total gyro ~= 0 deg/s, true
+    gyro_is_stationary = norm(gyro) < 1
+
+    # print result to user
+    if gyro_is_stationary:
+        print("Stationary gyroscope reading is near 0 deg/s.")
+    else:
+        print("Stationary gyroscope reading is not near 0 deg/s.")
+
+    # update result dictionary
+    result_dict["IMU_GyroStationary"] = (gyro_string, gyro_is_stationary)
+    return result_dict
+
+
+def rotating_imu_test(result_dict):
+    """
+    Return acceleration, gyro, and magnetic readings from request_imu_data
+    If None, update result dictionary that test was not run
+    Else, check that the norm of the average gyroscope reading is positive
+    and update result dictionary
+    """
+    # record gyro reading
```

Comment:
Same thing about unnecessary comments, `\n` in prompt, and just returning a `None`